main.py

```diff
@@ -1,12 +1,224 @@
-import requests
+from rentals import RentalList
+from videos import VideoList
+from customers import CustomerList
 
 URL = "http://127.0.0.1:5000"
 BACKUP_URL = "https://retro-video-store-api.herokuapp.com"
 
 def main():
     print("WELCOME TO RETRO VIDEO STORE")
-    pass
+    
+
+def print_stars():
+    print("\n**************************\n")
+
+def list_options():
+
+    options = {
+        "1": "List all videos", 
+        "2": "Create a new video",
+        "3": "Select a single video", 
+        "4": "Update a video", 
+        "5": "Delete a video", 
+        "6": "List all customers",
+        "7": "Create a new customer",
+        "8": "Select a single customer",
+        "9": "Update a customer",
+        "10": "Delete a customers",
+        "11": "Check OUT a video to a customer",
+        "12": "Check IN a video from a customer",   
+        "13": "List all options",
+        "14": "Quit"
+        
+        }
+
+    print_stars()
+    print("Welcome to the Video Store CLI")
+    print("These are the actions you can perform")
+    print_stars()
+    
+    for choice_num in options:
+        print(f"Option {choice_num}. {options[choice_num]}")
+
+    print_stars()
+
+    return options
+
+
+def make_choice(options, video_list, customer_list, rental_list):
+    valid_choices = options.keys()
+    choice = None
+
+    while choice not in valid_choices:
+        print("What would you like to do? Select 13 to see all options again")
```

Comment:
I like how you gave the user the option to see the options again, super helpful!